front-end/src/utils/NotifyHeader.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import styled from "styled-components";
+import { StyledHeader } from "./styles";
+import { ColoredButton } from "../AdminBar/adminStyles";
+import { NotifyContainer, FormContainer, WaitInput } from "./styles";
+import bell from "./bell.png";
+
+const BELL = 0;
```

Comment:
WDYT about having all these states in an object, e.g.

```
const STATE = {
  BELL: 0,
  INPUT: 1,
  /* and so on */
};
```

Reply:
done!